### DISCUSSION TITLE
MicroStream Cluster?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [microstream-one/microstream#621] [General]
What are the plans for [MicroStream Cluster](https://microstream.one/products/microstream-cluster/)? Is it going to be open-source and (eventually) migrated to the Eclipse foundation? Any timelines?

Thanks!

**Comment by fh-ms** [comment]
MicroStream Cluster is a closed-source enterprise product. There are currently no plans to open-source it.

**Reply by hrstoyanov** [reply]
Ok, but please make this VERY clear  on the web site!
